qcodes/dataset/data_set.py

```diff
@@ -1693,6 +1695,29 @@ def _flush_data_to_database(self, block: bool = False) -> None:
             log.debug(f"Waiting for write queue to empty.")
             writer_status.data_write_queue.join()
 
+    def _export_data_as_netcdf(self, path: str, prefix: str) -> None:
+        from qcodes.dataset.data_export import DataExportType
+        extension = DataExportType.NETCDF.value
+        path = os.path.join(path, f"{prefix}{self.run_id}.{extension}")
+        xarr_dataset = self.to_xarray_dataset()
+        xarr_dataset.to_netcdf(path=path)
+
+    def export_data(self) -> None:
+        """Export data to disk with file name {prefix}{run_id}.{ext}.
+        Values for the export type, path and prefix are set in the qcodes "dataset" config.
+        """
+
+        from qcodes.dataset.data_export import DataExportType, get_data_export_type, get_data_export_path, get_data_export_prefix
+        data_export_type = get_data_export_type()
```

Comment:
Personally I would make these input arguments too and then use the config iff no input argument is set